### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by oleksandra-lavryk**

### CODE REVIEW

File: javascript/javascript1/week2/main.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    }
+  }
+  return firstname + " " + surname;
+}
+
+const fullname1 = getFullname("Benjamin", "Hughes", true, "male");
+const fullname2 = getFullname("Line", "Hansen", true, "female");
+
+console.log(fullname1 + "\n" + fullname2);
+
```
Comment (vulk29) [inline javascript/javascript1/week2/main.js:28]: this looks good, but you forgot a condition>
'Remember to consider someone calling the function with an empty string as firstname and lastname' 
you could e.g. add a warning?

File: javascript/javascript1/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    todayFullDate.getFullYear(),
+    todayFullDate.getMonth(),
+    todayFullDate.getDate() + daysToEvent
+  );
+  return weekDays[meetingDate.getDay()];
+}
+
+console.log(getEventWeekday(5));
+console.log(getEventWeekday(2));
+
```
Comment (vulk29) [inline javascript/javascript1/week2/main.js:52]: good job!

File: javascript/javascript1/week2/main.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  } else if (temperature < 20) {
+    return "t-shirt and jeans";
+  } else {
+    return "shorts and t-shirt";
+  }
+}
+
+const clothesToWear = getClothToWear(20);
+console.log(clothesToWear);
+
```
Comment (vulk29) [inline javascript/javascript1/week2/main.js:70]: nice! well done with the else ifs!

File: javascript/javascript1/week2/main.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+addStudentToClass("Sasha");
+addStudentToClass("Pasha");
+addStudentToClass("Vera");
+addStudentToClass("Sara");
+addStudentToClass("Line");
+addStudentToClass("Ben");
+addStudentToClass("Pasha");
+addStudentToClass("Queen");
+console.log(getNumberOfStudents());
+
```
Comment (vulk29) [inline javascript/javascript1/week2/main.js:100]: good job here with the logic of else ifs!

File: javascript/javascript1/week2/main.js [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+    }
+  } else {
+    console.log("No money.Enough candy for you!");
+  }
+}
+
+addCandy("sweet", 20);
+addCandy("toffee", 10);
+addCandy("chewing-gum", 50);
+console.log(boughtCandyPrices);
```
Comment (vulk29) [inline javascript/javascript1/week2/main.js:148]: This is almost there! 🍡 
if i try addCandy("", 20);  i get: 
Unknown type of candy.
You can buy more, so please do!
[]

so maybe you can add a special log for unknown candy - e.g. 'we dont have that, but try buying sweet, toffee or chewing gum'? 
It is not essential for this _optional_ exercise, but the trick is to think through all the cases :)

**Review by vulk29 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week2/main.js (hunk 1 above)
Comment (oleksandra-lavryk) [reply javascript/javascript1/week2/main.js:28]: @vulk29 thanks, ofcourse I will add this condition))

File: javascript/javascript1/week2/main.js (hunk 2 above)
Comment (oleksandra-lavryk) [reply javascript/javascript1/week2/main.js:52]: @vulk29 Thanks a lot))

File: javascript/javascript1/week2/main.js (hunk 1 above)
Comment (vulk29) [reply javascript/javascript1/week2/main.js:28]: nice!! :)

**Pull request merged**